src/main/java/springboot/Service/PersonService.java

```diff
@@ -17,4 +17,6 @@
     public void addPerson(Person person){
         personRepository.save(person);
     }
+
+    public Person getPerson(String name) {return personRepository.findByName(name);}
```

Comment:
change to 
public Person getPerson(Integer id) {return itemRepository.findById(id);}

Reply:
go you mean personrepository.findbyid